.drone.yml

```diff
@@ -28,7 +28,18 @@ steps:
       - python3 tools/GenerateChangelog/ss13_genchangelog.py html/changelog.html html/changelogs --dry-run
       - python3 tools/mapmerge2/travis_mapcheck.py
       - echo "6a5ae087fe5bfa66e52e508655e57120 *html/changelogs/example.yml" | md5sum -c -
-      - echo "Complated Code Checks"
+      - echo "Completed Code Checks"
+  - name: "Run DreamChecker"
+    image: aurorastation/dm-buildimage:latest
+    commands:
+      - /opt/dreamchecker
+  - name: "Vue - Lint and Compile"
+    image: node:10-buster
+    commands:
+      - cd vueui
+      - npm install # TODO: Cache that shit
```

Comment:
This may be usefull. https://laszlo.cloud/the-ultimate-droneci-caching-guide

Reply:
I am aware of that, but we dont have a S3 storage yet. 
And the volume cache is out of the question as it needs to be run in privileged mode.